backend/resolvers/mutations/entry.js

```diff
@@ -61,7 +61,7 @@ const createEntry = (entry, entryType, entryId, t) => {
           )
         }
 
-        else if (user.displayName != entry.displayName){
+        else if (!group && user.displayName != entry.displayName){
```

Comment:
This should ensure that it only checks for a hometown and display name on single user submissions.